multihost_test/bz_automation/conftest.py

```diff
@@ -0,0 +1,54 @@
+"""conftest.py for authselect"""
```

Comment:
The code in this file is almost the same as in `multihost_test/Sanity/conftest.py`. Can't you reuse it?

Reply:
that way IDM CI does not work , you can see sssd repo for the same , every test directly has its own conftest.py file